cockroachcloud/frequently-asked-questions.md

```diff
@@ -115,16 +117,26 @@ Contact [Support](https://support.cockroachlabs.com/hc/en-us) to change your clu
 
 You can add nodes by accessing the **Clusters** page on the [{{ site.data.products.db }} Console](https://cockroachlabs.cloud/) and clicking the **...** button for the cluster you want to add or delete nodes for. See [Cluster Mangement](cluster-management.html?filters=dedicated#add-or-remove-nodes-from-a-cluster) for more details.
 
+During [limited access](/docs/{{site.versions["stable"]}}/cockroachdb-feature-availability.html), {{ site.data.products.dedicated }} clusters on Azure cannot be modified or scaled after creation. Refer to [{{ site.data.products.dedicated }} on Azure](cockroachdb-dedicated-on-azure.html).
+
 {% include cockroachcloud/nodes-limitation.md %}
 
 ### Do you auto-scale?
 
-Today, we do not automatically scale nodes based on your capacity usage. To add or remove nodes, see [Cluster Mangement](cluster-management.html?filters=dedicated#add-or-remove-nodes-from-a-cluster). There are plans to allow auto-scaling in the future.
+Today, we do not automatically scale nodes based on your capacity usage. To add or remove nodes, see [Cluster Mangement](cluster-management.html?filters=dedicated#add-or-remove-nodes-from-a-cluster).
+
+During [limited access](/docs/{{site.versions["stable"]}}/cockroachdb-feature-availability.html), {{ site.data.products.dedicated }} clusters on Azure cannot be modified or scaled after creation. Refer to [{{ site.data.products.dedicated }} on Azure](cockroachdb-dedicated-on-azure.html).
```

Comment:
We use this snippet so often throughout this document, consider pulling it out to its own include file.

Reply:
Actually the specific messaging is not the same everywhere -- this one is about modification and scaling, but others are about other specific limitations, which will need to be addressed separately as they are lifted. I agree about reuse where possible.